src/axom/core/examples/core_numerics.cpp

```diff
@@ -53,11 +53,15 @@ void demoVectorOps()
   double v[] = {1., 2., 3.};
   double w[] = {0., 0., 0.};
 
+  std::cout << std::endl;
```

Comment:
Side thought:
I know that `core` cannot depend on `slic` (since `slic` depends on `core`), but is there any reason not to have `core`'s examples depend on `slic`?

Reply:
I don't think that would be an issue. I didn't write the examples, just made minor mods to the code to make the output easier to read.